src/components/InfoTip/InfoTip.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown } from '../Dropdown';
+
+class InfoTip extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
```

Comment:
we have `bindMethods` helpers for these...would be nice to be consistent. [example](https://github.com/patternfly/patternfly-react/blob/master/src/components/Table/__mocks__/mockClientPaginationTable.js#L60)